### DISCUSSION TITLE
What is the journal feature for?

### DISCUSSION THREAD

**Discussion by bobosola** [zed-industries/zed#6539] [Feature Requests]
Probably a stupid question, but I just discovered the journal feature and don't really know what it's for? It appears to default to adding a new page by year, month and day with a new heading for local time each time you open it. So I'm guessing it's maybe meant for keeping a progress log of sorts? It's not something I've seen before in a code editor before. Does it magically link in to code files somehow?

**Comment by JosephTLyons** [comment]
The journal feature is really just as simple as it seems - it doesn't do anything fancy with linking content.  It was added very early on, back in 2021, before Zed was even in private alpha.  I believe the founders baked it in as quick way to take notes while in meetings.  I don't know what our plan going forward is, but I could see us demoting it to a plugin, once we have support for plugins.

**Reply by avetisk** [reply]
I think it's a nice feature! But indeed, it might be better as a plugin.

**Comment by goodlux** [comment]
I love the journal feature! One of my favorite things about Zed! Having it as part of the core reminds me that it's not just about code, it's about creation. 

Oh no ... just checked ... is it GONE? 

Ah well ... I guess if you are going to go minimal, you go minimal!

**Reply by narthur** [reply]
It's still there. In command palette, "journal: new journal entry."

**Reply by socketbox** [reply]
Also discovered, quite by accident, that typing `j` and hitting `Enter` will also do the same (it opens a new window and creates a new markdown file with a filename of the current date).

**Comment by xtrasmal** [comment]
Added this: https://github.com/zed-industries/zed/discussions/31889
